### ISSUE TITLE
Subtitles support

### ISSUE THREAD

**Issue by ghost** [codemanav/InfiMaze#1]
Would be nice to have subtitles.. maybe an opensubtitle integration?

**Comment by codemanav** [comment]
Yes, nice suggestion. I will be adding it soon. If you like you can also contribute to it.

**Comment by codemanav** [comment]
Pull Requests are always welcome.
